docs/in-app-messaging/usage/index.md

```diff
@@ -49,6 +49,10 @@ Firebase Console are automatically handled and displayed on your user's device.
 
 This module provides a JavaScript API to allow greater control of the displaying of these messages.
 
+# Limitations
+Accorrding to [Gihthub isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch - this is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (ie: Analytics event or problematically triggered FIAM campaigns), there is no daily rate limit.
```

Comment:
Do you mean `programmatically` triggered vs `problematically` ? I ask because with software you never know, there could be another issue causing campaign triggering to be problematic as well 😅